docs/pnl-fix-summary.md

```
@@ -0,0 +1,164 @@
+
+## Verification
+
+### Testing the Fix
+
+1. **API Test**: Call the portfolio endpoint for an agent with closed positions
+
+   ```bash
+   curl 'https://staging.babylon.market/api/npc/[actorId]/portfolio' \
+     -H 'Cookie: ...'
+   ```
+   
```

markdownlint-cli2 (0.20.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 3

(MD009, no-trailing-spaces)